library/base16_builder.py

```diff
@@ -612,7 +620,7 @@ def run(self):
             if len(scheme_result) == 1 and self.module.params["template"]:
                 failure_msg = "Failed to build any templates."
                 if self.module.params["template"]:
-                    failure_msg = '{} Template name "{}" was passed, but didn\'t match any known templates'.format(
+                    failure_msg = '{} Template names {} were passed, but didn\'t match any known templates'.format(
```

Comment:
It looks like this change is breaking a test at test_base16_builder.py line 649 which asserts on the failure message. You can see it in [travis](https://travis-ci.org/github/mnussbaum/base16-builder-ansible/builds/753671375). Mind updating the tests?

Reply:
Indeed, I forgot to run the automated tests. I've updated those and they run fine now.